packages/app/src/app/pages/Sandbox/Editor/Workspace/screens/Deployment/index.tsx

```diff
@@ -33,7 +33,7 @@ export const Deployment: FunctionComponent = () => {
     <Collapsible title="Deployment" defaultOpen>
       <Element paddingX={2}>
         <Text variant="muted" block marginBottom={6}>
-          You can deploy a production version of your sandbox using one our
+          You can deploy a production version of your sandbox using one of our
           supported providers.
```

Comment:
Bring in line with the other texts

```suggestion
          supported providers - Netlify or ZEIT.
```